models/taskRequests.js

```diff
@@ -171,10 +171,16 @@ const fetchTaskRequestById = async (taskRequestId) => {
     const taskRequestSnapshot = await taskRequestsCollection.doc(taskRequestId).get();
     const taskRequestData = taskRequestSnapshot.data();
 
-    if (taskRequestData) {
-      taskRequestData.id = taskRequestSnapshot.id;
-      taskRequestData.url = new URL(`/taskRequests/${taskRequestData.id}`, config.get("services.rdsUi.baseUrl"));
+    if (!taskRequestData) {
+      return {
+        taskRequestData,
```

Comment:
Can't we just return taskRequestExists: false? why to return additional null or undefined value.

Reply:
This is updated, thanks for the suggestion. I will be careful next time.